_pages/fr/participation.md

```diff
@@ -0,0 +1,36 @@
+---
+layout: default
+title:  "Participation"
+permalink: /lang/participation
```

Comment:
since this is the FR version, please already replace "lang" with "fr" (same for the lang: property below)